pkg/workloads/specjbb/jvm_options.go

```diff
@@ -23,9 +23,9 @@ import (
 
 var (
 	// JVMHeapMemoryGBs specifies amount of heap memory available to JVM.
-	JVMHeapMemoryGBs = conf.NewIntFlag("specjbb_jvm_heap_size", "Size of JVM heap memory in gigabytes", 2)
+	JVMHeapMemoryGBs = conf.NewIntFlag("specjbb_jvm_heap_size", "Size of JVM heap memory in gigabytes", 8)
```

Comment:
@skonefal - why have you decided to increase heap size and number of GC threads?